src/Function/Construct/Setoid.agda

```diff
@@ -0,0 +1,37 @@
+-- agda-categories says that it:
+-- was not ported from old function hierarchy
+
+{-# OPTIONS --without-K --safe #-}
+
+-- was not ported from old function hierarchy
+
+module Function.Construct.Setoid where
+
+  open import Function.Bundles using (Func)
+  import Function.Construct.Composition as Comp
+  open import Level using (Level)
+  open import Relation.Binary.Bundles using (Setoid)
+
+
+  private
+    variable
+      a₁ a₂ b₁ b₂ c₁ c₂ : Level
+
+  setoid : Setoid a₁ a₂ → Setoid b₁ b₂ → Setoid _ _
+  setoid From To = record
+    { Carrier = Func From To
+    ; _≈_ = λ f g → ∀ {x} → Func.to f x To.≈ Func.to g x
+    ; isEquivalence = record
+      { refl = To.refl
+      ; sym = λ f≈g → To.sym f≈g
+      ; trans = λ f≈g g≈h → To.trans f≈g g≈h
+      }
+    }
+    where
+      module To = Setoid To
+
+  -- This doesn't really belong here, it should be in Function.Construct.Composition but that's in stdlib
+  -- so will need to be contributed to there first.
```

Comment:
I don't understand this comment. Could you please rephrase?

Reply:
Oh, I took that file verbatim from [agda-categories](https://github.com/agda/agda-categories/blob/master/src/Function/Construct/Setoid.agda). What they mean is (I believe) that they wanted a nice infix operator for [Function.Construct.Composition.function](https://agda.github.io/agda-stdlib/v2.0/Function.Construct.Composition.html#5240) and therefore it should be in module `Function.Construct.Composition`, but that is in stdlib.

I think, that I didn't use this one, so perhaps we could just remove it?

Reply:
Thanks for the clarification! I don't usually give the categorical operators any name other than the standard one (in the raw `Category` class).